### PR TITLE
Config backend wasn't dockerizing script paths.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Comment by kshakir** [comment]
Fixes the workaround syntax in #1126, but doesn't completely repair the issue.

File: core/src/main/resources/reference.conf [hunk 1]
```diff
@@ -179,7 +179,14 @@ backend {
         String? docker_user
         """
         submit = "/bin/bash ${script}"
-        submit-docker = """docker run --rm ${"--user " + docker_user} -v ${cwd}:${docker_cwd} -i ${docker} /bin/bash ${docker_cwd}/execution/script"""
+        submit-docker = """
+        docker run \
+          --rm -i \
+          ${"--user " + docker_user} \
+          -v ${cwd}:${docker_cwd} \
+          ${docker} \
```
Comment (mcovarr) [inline core/src/main/resources/reference.conf]: ToL I kinda wish this variable name had the word "image" in it for clarity, but unless we support a new and deprecated name for this variable that would be a breaking change 🙁

File: supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigAsyncJobExecutionActor.scala [hunk 2]
```diff
@@ -64,23 +64,28 @@ sealed trait ConfigAsyncJobExecutionActor extends SharedFileSystemAsyncJobExecut
...
+      CwdInput -> WdlString(jobPaths.callRoot.pathAsString)
     )
   }
 
   /**
-    * Extra arguments if this is a submit-docker command, or Map.empty.
+    * The inputs that are not specified by the config, that will be passed into a command for either submit or
+    * submit-docker.
     */
   private lazy val dockerInputs: WorkflowCoercedInputs = {
```
Comment (mcovarr) [inline supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigAsyncJobExecutionActor.scala]: The method name doesn't seem quite right anymore.

**Comment by mcovarr** [comment]
:+1:

**Comment by cjllanwarne** [comment]
ToL: Is this centaurable? Or can we reconfigure centaur's Local tests so that this is verified?

**Comment by cjllanwarne** [comment]
👍 not sure how docker was working at all in SFS-land before this!

**Comment by kshakir** [comment]
The `script` variable was busted, but docker was working in SFS land because, instead of `${script}` the local backend was using `${cwd}/execution/script` when launching docker jobs.

File: supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigAsyncJobExecutionActor.scala (hunk 2 above)
Comment (kshakir) [inline supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigAsyncJobExecutionActor.scala]: Naming... my nemesis.

- `dockerizedInputs`
- `localOrDockerInputs`
- `submitOrSubmitDockerInputs`
- etc.?

**Comment by kshakir** [comment]
Re "centaurable"-- I'm confident by updating the local backend to use `${script}` that centaur is actually exercising a third of the variables updated in patch. We could also change centaur to completely run on a new backend that also uses the `${out}` and `${err}` paths, as outlined in issue #1126. But because of that issue, this new centaur test would require a separate, new backend definition in `local_centaur.conf`.

FYI: I still don't have a full fix for #1126, but this is a step in that direction. When I looked at the reference.conf, I noticed the local backend wasn't using `${script}`, wanted to know why, and discovered this small issue. This patch will also make the workaround in that ticket work as expected, instead of pointing to paths outside the docker container like `/Users/kshakir/<path>` now pointing paths inside like `/cromwell_root/<path>`.

File: core/src/main/resources/reference.conf (hunk 1 above)
Comment (kshakir) [inline core/src/main/resources/reference.conf]: Not touching that in this PR, but open to one filing a ticket to revisit!

File: supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigAsyncJobExecutionActor.scala (hunk 2 above)
Comment (kshakir) [inline supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigAsyncJobExecutionActor.scala]: Ruchi hath selected `dockerizedInputs`. Updating.

**Pull request merged**